Flippers/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+}
+
+
+android {
+    setCompileSdkVersion(30)
+
+    defaultConfig {
+        setMinSdkVersion(16)
+        targetSdkVersion(30)
+        versionCode = 1
+        versionName = "1.0"
+    }
+}
+
+plugins.withId("java") {
+    configure<JavaPluginExtension> {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    tasks.withType<JavaCompile> {
+        options.encoding = "UTF-8"
+    }
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    kotlinOptions {
+        freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+}
+
+dependencies {
+    implementation("androidx.recyclerview:recyclerview:1.1.0")
+    api("androidx.appcompat:appcompat:1.2.0")
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.2")
+    api("androidx.lifecycle:lifecycle-runtime-ktx:2.3.0")
+    api("androidx.core:core-ktx:1.3.2")
```

Comment:
I'm removing all the unnecessary libs. The lifecycle will be important because I'm implementing lifecycle observing in order to not retain things after the view is destroyed.